Touch Bar Simulator/TouchBarWindow.swift

```diff
@@ -11,8 +11,10 @@ final class TouchBarWindow: NSPanel {
 				window.addTitlebar()
 			case .dockedToTop:
 				window.removeTitlebar()
+				window.roundCorners([.layerMaxXMinYCorner, .layerMinXMinYCorner])
```

Comment:
Add a couple of `CACornerMask` extensions to make this and the below call nicer. `.minY` and `.maxY`.